src/lsdb/loaders/dataframe/dataframe_catalog_loader.py

```diff
@@ -112,20 +110,32 @@ def _calculate_threshold(self, partition_size: int | None = None, threshold: int
         return threshold
 
     @staticmethod
-    def _create_catalog_info(**kwargs) -> CatalogInfo:
+    def _create_catalog_info(
+        catalog_name: str = "from_lsdb_dataframe",
+        ra_column: str = "ra",
+        dec_column: str = "dec",
+        catalog_type=CatalogType.OBJECT,
+        **kwargs,
+    ) -> TableProperties:
         """Creates the catalog info object
 
         Args:
+            catalog_name: it is recommended to provide a new name for your catalog
+            ra_column: column to find right ascension coordinate
+            dec_column: column to find declination coordinate
+            catalog_type: type of table being created (e.g. object, margin, index)
```

Comment:
```suggestion
            catalog_type: type of table being created (e.g. OBJECT, MARGIN, INDEX)
```
To signal that these are enumeration values.  I do see that the enumeration is based on `str` and that the string values of those enumerations are indeed the lowercase names, so this is a very light suggestion.